cpp/search/searchresults.cpp

```diff
@@ -1311,3 +1310,144 @@ double Search::getAverageTreeOwnershipHelper(vector<double>& accum, int64_t minV
 
   return desiredWeight;
 }
+
+json Search::getJsonOwnershipMap(const Player pla, const Player perspective, const Board& board, const SearchNode* node, int ownershipMinVisits) const {
+  vector<double> ownership = getAverageTreeOwnership(ownershipMinVisits, node);
+  json ownerships = json::array();
+  for(int y = 0; y < board.y_size; y++) {
+    for(int x = 0; x < board.x_size; x++) {
+      int pos = NNPos::xyToPos(x, y, nnXLen);
+      double o;
+      if(perspective == P_BLACK || (perspective != P_BLACK && perspective != P_WHITE && pla == P_BLACK))
+        o = -ownership[pos];
+      else
+        o = ownership[pos];
+      ownerships.push_back(o);
+    }
+  }
+  return ownerships;
+}
+
+bool Search::getAnalysisJson(
+  const Player pla,
+  const Player perspective,
+  const Board& board,
+  const BoardHistory& hist,
+  int minMoves,
+  int analysisPVLen,
+  int ownershipMinVisits,
+  bool preventEncore,
+  bool includePolicy,
+  bool includeOwnership,
+  bool includeMovesOwnership,
+  bool includePVVisits,
+  json& ret) const {
+  vector<AnalysisData> buf;
+  getAnalysisData(buf, minMoves, false, analysisPVLen);
+
+  // Stats for all the individual moves
+  json moveInfos = json::array();
+  for(int i = 0; i < buf.size(); i++) {
+    const AnalysisData& data = buf[i];
+    double winrate = 0.5 * (1.0 + data.winLossValue);
+    double utility = data.utility;
+    double lcb = 0.0; //    PlayUtils::getHackedLCBForWinrate(this, data, pla);
```

Comment:
What is the reason for commenting this out?

Reply:
Some header conflict which led to a 'i'll fix this later oversight'